src/win/udp.c

```diff
@@ -189,6 +189,11 @@ void uv_udp_endgame(uv_loop_t* loop, uv_udp_t* handle) {
 }
 
 
+int uv_udp_is_using_recvmmsg(const uv_udp_t* handle) {
+  return !!(handle->flags & UV_HANDLE_UDP_RECVMMSG);
+}
```

Comment:
Also, since the implementation is the same on Windows and Unix, I think this could be placed in the common code.

Reply:
Same thing here, them being separated matches the existing flag getters:

https://github.com/libuv/libuv/blob/e7ebae26247d2fee0a04547eb7f9aa8f78d4a642/src/win/stream.c#L223-L230

https://github.com/libuv/libuv/blob/e7ebae26247d2fee0a04547eb7f9aa8f78d4a642/src/unix/stream.c#L1606-L1613